src/core/basic_client.rs

```diff
@@ -722,6 +724,95 @@ impl BasicClient {
         Ok(())
     }
 
+    /// **[Cryptographic Operation]** Create an asymmetric signature on a message.
+    ///
+    /// The key intended for signing **must** have its `sign_hash` flag set
+    /// to `true` in its [key policy](https://docs.rs/parsec-interface/*/parsec_interface/operations/psa_key_attributes/struct.Policy.html).
+    ///
+    /// The signature will be created with the algorithm defined in
+    /// `sign_algorithm`, but only after checking that the key policy
+    /// and type conform with it.
+    ///
+    /// # Errors
+    ///
+    /// If the implicit client provider is `ProviderId::Core`, a client error
+    /// of `InvalidProvider` type is returned.
+    ///
+    /// See the operation-specific response codes returned by the service
+    /// [here](https://parallaxsecond.github.io/parsec-book/parsec_client/operations/psa_sign_hash.html#specific-response-status-codes).
```

Comment:
I guess the link should be `https://parallaxsecond.github.io/parsec-book/parsec_client/operations/psa_sign_message.html#specific-response-status-codes`

Reply:
Corrected - thanks for pointing.